modules/system/classes/CombineAssets.php

```diff
@@ -338,19 +338,32 @@ protected function prepareAssets(array $assets)
         /*
          * Determine which group of assets to combine.
          */
-        if (count($combineCss) > count($combineJs)) {
+        $cssCount = count($combineCss);
+        $jsCount = count($combineJs);
+        if ($cssCount > $jsCount) {
             $extension = 'css';
             $assets = $combineCss;
         }
-        else {
+        elseif ($cssCount < $jsCount) {
             $extension = 'js';
             $assets = $combineJs;
         }
+        else {
+            $extension = null;
+            $assets = array_unique([...$combineCss, ...$combineJs]);
```

Comment:
What case does this cover?

Reply:
The actual case I try to solve actually:
Using at least one css not-aliased file, the if branch will match.
Using at least one js not-aliased file, the else if branch will match.
Using only aliased file(s) or an equality of css and js + aliases, both css and js stacks counts are equal, so I merge all of them with a unique rule (because both framework.extra exists as a css alias and js alias).

https://github.com/wintercms/winter/blob/6e8c8d90f11cf38560f3e5f1f1fe3ffac904086c/modules/system/classes/CombineAssets.php#L160-L163